Remotion/Core/Reflection.UnitTests/TypeAdapterTest.cs

```diff
@@ -80,14 +80,14 @@ public void FullName ()
     public void Namespace ()
     {
       var type = typeof (ArrayList);
-      Assert.That (TypeAdapter.Create (type).Namespace, Is.EqualTo (type.Namespace));
+      Assert.That (TypeAdapter.Create (type).Namespace, Is.EqualTo (type.GetNamespaceChecked()));
```

Comment:
Always revert in tests
```suggestion
      Assert.That (TypeAdapter.Create (type).Namespace, Is.EqualTo (type.Namespace));
```

Reply:
All of the extension methods or just the namespace?

Reply:
all extension methods: reasoning: in tests we should always have "correct" values when working with assertions so we don't need the helper methods.